op-node/rollup/sequencing/sequencer.go

```diff
@@ -179,7 +242,7 @@ func (d *Sequencer) handleInvalid() {
 }
 
 func (d *Sequencer) onInvalidPayloadAttributes(x engine.InvalidPayloadAttributesEvent) {
-	if x.Attributes.DerivedFrom == (eth.L1BlockRef{}) {
+	if x.Attributes.DerivedFrom != (eth.L1BlockRef{}) {
```

Comment:
Well damn that would have been a nasty thing to miss.  Maybe we should have a `isDerivedFromL1` or `isFromSequencer` helper function just so there's a name that makes it easier to quickly see what the check does.  Otherwise it's really easy to see this check and not notice the difference between `!=` and `==` because you see what you want to see.

Reply:
The chaos test shakes out things like this, without it it will run out of events without scheduling a new sequencer action, due to improper error handling. If you flip it, rerun the chaos test, all 100 generated sub test-cases will very quickly fail. I'm confident we have the testing to catch cases like this. A helper function may be nice, but I think as-is is fine.